src/views/Swap/Swap.vue

```diff
@@ -566,6 +567,10 @@ export default {
     amountError () {
       const amount = BN(this.safeAmount)
 
+      if (this.showNoLiquidityMessage === true) {
```

Comment:
better written `if (this.showNoLiquidityMessage)`

Reply:
also should be at the top of the function since nothing needs to happen if it returns